Backend/src/test/java/com/senla/repository/EventRepositoryTest.java

```diff
@@ -0,0 +1,27 @@
+package com.senla.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.senla.model.Event;
+import com.senla.model.User;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+
+class EventRepositoryTest extends DatabaseTest {
+
+    @Autowired private UserRepository userRepository;
+
+    @Autowired private EventRepository eventRepository;
+
+    @Test
+    void findByAuthorId() {
```

Comment:
For the test naming I suggest to use concept of GIVEN->WHEN->THEN 

Aslo it's good to have negative case when we can't find author by provided id.

Then test name will be somthing like:
- givenExistingAuthorId_whenFindingById_thenReturnAuthor - for the positive case
- givenNonExistingAuthorId_whenFindingById_thenReturnEmpty - for the negative case

 If this is the service layer then we can throw a NotFoundException so the naming will be;
- givenNonExistingAuthorId_whenFindingById_thenThrowNotFoundException